src/Hypernode/Magento/Command/Hypernode/Performance/PerformanceCommand.php

```diff
@@ -175,176 +188,256 @@ protected function getEffectiveUrl($url)
         return $tables;
     }
 
 
     /**
      * Generates data to output in a table.
+     *
      * @todo make table nicer, fix colorized output in diffferent format - new function
+     *
      * @param $results
+     *
      * @return array|bool
      */
     protected function generateTablesData($results)
     {
-        $tables = array(); // all tables
-        foreach ($results as $set) { // foreach sitemap we parsed
-
-            // setting placeholder var and headers
-            $tableArray = array(
-                'headers' => false,
-                'requests' => array()
-            ); // every table row
-
-            // setting headers
-            if (count($set[0]) > 1) {
-                $tableArray['headers'] = array("URL", "Status X", "TTFB X", "Status Y", "TTFB Y", "Difference");
-            } elseif (count($set[0]) == 1) {
-                $tableArray['headers'] = array("URL", "Status", "TTFB");
-            }
+        $tables = []; // all tables
+        // setting placeholder var and headers
+        $tableArray = [
+            'headers'  => false,
+            'requests' => [],
+        ]; // every table row
+
+        // setting headers
+        if ($this->_options['compare-url']) {
+            $tableArray['headers'] = [
+                "URL",
+                "Status X",
+                "TTFB X",
+                "Status Y",
+                "TTFB Y",
+                "Difference",
+            ];
+        } else {
+            $tableArray['headers'] = ["URL", "Status", "TTFB"];
+        }
 
-            foreach ($set as $batch) {
+        foreach ($results as $result) { // foreach sitemap we parsed
 
-                $requestArray = array();
+            $requestArray = [];
 
-                if (count($batch) > 1) {
-                    $parsedUrl = parse_url($batch[0]['url']);
-                    $requestArray[] = $parsedUrl['path'];
-                    foreach ($batch as $request) {
-                        $requestArray[] = $this->parseResponseCode($request['status']);
-                        $requestArray[] = $request['ttfb'];
-                    }
-                    $requestArray[] = $this->ttfbCompare($batch[0]['ttfb'], $batch[1]['ttfb']);
-                } elseif (count($batch) == 1) {
+            $parsedUrl = parse_url($result['current']['url']);
 
-                    $requestArray[] = $batch[0]['url'];
-                    $requestArray[] = $this->parseResponseCode($batch[0]['status']);
-                    $requestArray[] = $batch[0]['ttfb'];
-                } else {
-                    return false; // output data as json or something
-                }
-                array_push($tableArray['requests'], $requestArray);
+            if ($this->_options['compare-url']) {
+                $requestArray[] = $parsedUrl['path'];
+            } else {
+                $requestArray[] = $result['current']['url'];
+            }
+
+
+            //$requestArray[] = $result['current']['url'];
```

Comment:
This commented line can be removed right?

Reply:
Yes, I've removed it.